CMakeLists.txt

```diff
@@ -776,6 +776,9 @@ if(opencoarrays_aware_compiler)
   add_caf_test(issue-488-multi-dim-cobounds-true  8 issue-488-multi-dim-cobounds true)
   add_caf_test(issue-488-multi-dim-cobounds-false 8 issue-488-multi-dim-cobounds false)
   add_caf_test(issue-503-multidim-array-broadcast 2 issue-503-multidim-array-broadcast)
+  if((gfortran_compiler AND (NOT CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 7.0.0)) OR (CAF_RUN_DEVELOPER_TESTS OR $ENV{OPENCOARRAYS_DEVELOPER}))
+    add_caf_test(issue-515-mimic-mpi-gatherv 2 issue-515-mimic-mpi-gatherv)
+  endif()
```

Comment:
Great, thanks! (Guard needed because functionality of allocatable components missing in GFortran < 7.1)